src/main/kotlin/no/nav/tpts/mottak/clients/TokenXClient.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.tpts.mottak.clients
+
+import com.nimbusds.jose.JOSEObjectType
+import com.nimbusds.jose.JWSAlgorithm
+import com.nimbusds.jose.JWSHeader
+import com.nimbusds.jose.crypto.RSASSASigner
+import com.nimbusds.jose.jwk.RSAKey
+import com.nimbusds.jwt.JWTClaimsSet
+import com.nimbusds.jwt.SignedJWT
+import io.ktor.client.request.forms.submitForm
+import io.ktor.client.request.get
+import io.ktor.http.Parameters
+import kotlinx.coroutines.runBlocking
+import no.nav.security.token.support.client.core.OAuth2GrantType
+import java.time.Instant
+import java.util.*
+
+private val wellKnown = System.getenv("TOKEN_X_WELL_KNOWN_URL")
+private val clientId = System.getenv("TOKEN_X_CLIENT_ID")
+private val privateJwk = System.getenv("TOKEN_X_PRIVATE_JWK")
+private const val tokenXAudience = "https://tokendings.dev-gcp.nais.io/token"
```

Comment:
Detekt sier
Top level property names should follow the naming convention set in the projects configuration.
Warning
Top level constant names should match the pattern: [A-Z][_A-Z0-9]*